File: programmers/탐욕법(Greedy)/단속카메라.js
function solution(routes) {
  let answer = 0;
  routes.sort((a, b) => a[1] - b[1]);
  let cam_pos = -30000;
  for (const [r_s, r_e] of routes) {
    if (cam_pos < r_s) {
      cam_pos = r_e;
      answer++;
    }
  }
  return answer;
}

File: programmers/탐욕법(Greedy)/단속카메라.py
def solution(routes):
    answer = 0
    routes.sort(key=lambda x: x[1])
    cam_pos = -30000
    for r_s, r_e in routes:
        if cam_pos < r_s:
            cam_pos = r_e
            answer += 1
    return answer
